airflow/providers/microsoft/azure/hooks/azure_functions.py

```diff
@@ -0,0 +1,177 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from __future__ import annotations
+
+from typing import Any
+
+import requests
+from azure.identity import ClientSecretCredential
+from requests import Response
+
+from airflow.exceptions import AirflowException
+from airflow.hooks.base import BaseHook
+
+
+class AzureFunctionsHook(BaseHook):
```

Comment:
Maybe my question dumb. I wondering is it possible to reuse AzureBaseHook?

https://github.com/apache/airflow/blob/2f552233f5c99b206c8f4c2088fcc0c05e7e26dc/airflow/providers/microsoft/azure/hooks/base_azure.py#L27

And is it better to use some kind of SDK rather than build HTTP requests manually? For example [`azure-functions`](https://github.com/Azure/azure-functions-python-library)

Reply:
I did check on this SDK, it supports to create of azure-function, deploy it how will you invoke the azure function which already exists?

Reply:
I am using same functionality, by building the HTTP requests manually.

Reply:
Unfortunately no SDK exists for Azure Functions specifically 😞